.gitignore

```diff
@@ -118,3 +118,14 @@ venv.bak/
 
 #sphinx build
 /docs/_build/
+/docs/build
+
+# Downloaded datasets for docs
+/docs/source/notebooks/aclImdb
+/docs/source/notebooks/data
+/docs/source/notebooks/*.tar.gz
+/docs/source/notebooks/spoken_digits
+/docs/source/notebooks/pretrained_models
```

Comment:
Is it better to store this stuff in s3?

Reply:
These are datasets that are downloaded when we run the tutorials in jupyter notebooks. I've included them here so that they don't get pushed to the remote repo.